draft-ietf-quic-recovery.md

```diff
@@ -245,6 +245,13 @@ QUIC specifies a time-based definition to ensure one or more packets are sent
 prior to a dramatic decrease in congestion window; see
 {{persistent-congestion}}.
 
+### Increase the min congestion window to 2 packets
```

Comment:
Use "minimum congestion window" to stay consistent with rest of the draft. FWIW RFC 5681 for TCP refers to this as loss window LW. Not sure if we want to use the same term in this draft.

Reply:
```suggestion
### The Minimum Congestion Window is 2 Packets
```

Title Case is Used in This Document